hloc/extract_features.py

```diff
@@ -66,6 +66,12 @@ class ImageDataset(torch.utils.data.Dataset):
         'grayscale': False,
         'resize_max': None,
     }
+    if platform.system() =='Windows':
+        default_conf = {
+            'globs': ['*.jpg', '*.png', '*.jpeg'],
+            'grayscale': False,
+            'resize_max': None,
+        }
```

Comment:
AFAIU glob is case-insensitive on Windows so `*.png` and `*.PNG` return identical lists. It would be easier to simply remove duplicates in `self.paths`, e.g. with
`self.paths = sorted(list(set(self.paths)))` (sorting to ensure deterministic)

Reply:
Great simple solution!! It works on my Windows system.